theurgy.lic

```diff
@@ -55,6 +55,14 @@ def buy_wine
   put_in_container('wine')
 end
 
+def buy_Taffelberries
+  DRCI.order_item(8280, 14)
+  fput 'get Taffelberries'
+  put_in_container('Taffelberries')
+  fput 'get Taffelberries'
+  put_in_container('Taffelberries')
+end
+
```

Comment:
order_item in common-items ends up buying 2 of these so I just get them both.  (will make you have to do this run less often)